src/Symfony/Component/Messenger/Tests/DependencyInjection/MessengerPassTest.php

```diff
@@ -413,3 +425,7 @@ public function handle($message, callable $next)
         return $next($message);
     }
 }
+
+final class MyTypeHintedBus extends DecoratedMessageBus
```

Comment:
TypeHinted is misleading here :) a class is a type, method arguments are type-hinted